cdap-ui/app/features/hydratorplusplus/leftpanel.less

```diff
@@ -223,26 +245,30 @@ body.theme-cdap.state-hydratorplusplus-create {
         background-color: white;
         border: 1px solid transparent;
         color: #666;
-        line-height: 40px;
+        line-height: 30px;
         margin-bottom: 6px;
         padding: 0 8px;
-        height: 40px;
+        height: 30px;
         width: 48%;
+        border: 1px solid #dddddd;
         .border-radius(4px);
         .transition(all 0.2s ease);
-        &.add-plugin-template:hover, &.add-plugin-template:focus, &:hover, &:focus {
-          background-color: fade(black, 45%);
-          border: 1px dashed #999;
-          color: white;
-        }
+        // &.add-plugin-template:hover, &.add-plugin-template:focus, &:hover, &:focus {
+        //   background-color: fade(black, 45%);
+        //   border: 1px dashed #999;
+        //   color: white;
+        // }
         &.add-plugin-template {
-          background: transparent;
-          border: dashed 1px;
+          width: 100%;
+          text-align: center;
+          background: #098BF9;
```

Comment:
Agreed. We need to move all the colors to be generic (used under a feature). Right now I have just used straight colors. Will do this post strata for 3.4 as it involves pulling out lot of variables out of a feature (atleast from hydrator++) and then reusing variables.